addons/base_setup/res_config_view.xml

```diff
@@ -82,6 +82,11 @@
                         </div>
                     </group>
                     <group>
+                        <label for="view_analytics_config_settings" string="Google Analytics"/>
+                        <div name="google_analytics">
```

Comment:
Why putting a void div here, and not putting your google analytics entry directly in website_version_views.xml using a correct xpath ? This makes more sense I think, base_setup should be as much module-independent as possible.